Sources/BraintreePayPal/BTPayPalVaultEditRequest.swift

```diff
@@ -5,13 +5,16 @@ import Foundation
 public class BTPayPalVaultEditRequest {
 
     private let editPayPalVaultID: String
+    public var merchantAccountID: String?
```

Comment:
```suggestion
    private let merchantAccountID: String?
```
I agree with Jax. Also any reason we can't make this a `data class`?

Reply:
Oh wow yeah I'm getting confused I thought this was Kotlin lol. Sorry. I forgot that Objective-C and Swift structs don't mix well. Plz disregard.

Reply:
No worries. You do bring up a good point about use of classes for requests.

Reply:
I think this could be a struct. For new features, we're trying to experiment w/ seeing if we get merchant feedback by not offering Objective-C support (ex: ShopperInsights & CreditMessaging don't support Obj-C). If we need to switch it to class later for ObjC bridging we can, but I like the idea of using a struct!